README.md

```diff
@@ -1,85 +1,131 @@
 
-Upon successful installation, verify the setup by executing the following command: `node -v`. If the installation was successful, this command should provide the version of Node currently installed
+To get linting in VSCode:
 
-### Installing `pnpm`
+1. Install the [ESLint Extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
+2. Always open the repo from its root. This allows the extension to detect the `.eslintrc.json` file which specifies the linter configuration.
+3. (Optional) Use the [VSCode workspace](./.vscode/Ottehr.code-workspace) for a helpful alternative organization of the project in the VSCode 'Explorer', which most developers find useful. This can be opened in 2 ways:
+   1. Open [the file](./.vscode/Ottehr.code-workspace) in VS Code. Click on the "Open Workspace" button in the bottom-right.
+   2. `code .vscode/Ottehr.code-workspace`
+
+### Node Installation
 
-The easiest way to get started is to use the brew command:
+To manage Node.js versions efficiently, we recommend using [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm#installing-and-updating).
+
+1. Install nvm by following the instructions provided [here](https://github.com/nvm-sh/nvm#installing-and-updating).
+
+2. Use nvm to install Node.js version 18 with the following commands:
+
+    ```bash
+    nvm install 18
+    ```
+
+3. Set Node.js version 18 as the default with:
+
+    ```bash
+    nvm alias default 18
+    ```
+
+After successful installation, verify the setup by executing:
 
 ```bash
-brew install pnpm
+node -v
 ```
 
-Otherwise, you can install it onto your machine using the [documentation](https://pnpm.io/installation).
-
-### Setup Script
+This command should display the installed Node.js version.
 
-The rest of this setup guide assumes that you have access to a ZapEHR project and have cloned down a fork of the ottehr repository. If you have not, please follow these steps:
+### Installing `pnpm`
 
-1. Fork Ottehr: https://github.com/masslight/ottehr/fork
-2. Copy your fork SSH clone link and `git clone git@github.com:{your_profile}/ottehr.git` in the folder you'd like to use
-3. (optional) Add ottehr as upstream `git remote add upstream git@github.com:masslight/ottehr.git`
-4. Open repo in editor of your choice, I will use vscode `code .vscode/Ottehr.code-workspace`
+To manage Node.js packages, we recommend using [pnpm](https://pnpm.io/).
 
-Before you continue, please ensure that you have [node](#node) v18.x and [pnpm](#installing-pnpm) installed on your machine.
+#### Using Homebrew (macOS/Linux):
 
-After you have these dependencies installed, please execute the setup script from the root directory:
+The easiest way to install `pnpm` is through Homebrew:
 
 ```bash
-sh scripts/setup.sh
+brew install pnpm
```

Comment:
We could probably change this to npm to be a bit more accessible.

```bash
npm install -g pnpm
```

Reply:
Will update that this afternoon